tests/integration/component.go

```diff
@@ -583,13 +583,10 @@ func componentTests(args ...string) {
 			ts := time.Now().UnixNano()
 			contextNumeric, err = ioutil.TempDir("", fmt.Sprint(ts))
 			Expect(err).ToNot(HaveOccurred())
-			os.Mkdir(context, 0750)
```

Comment:
Isn't this source of the original error? Instead of creating `context` dir wasn't this supposed to be `contextNumeric`?

Reply:
Yes, you are right. 
I thought global ```context``` is misplaced somehow here.

Reply:
I wrongly interpret your comment i guess.

@kadel ```contextNumeric, err = ioutil.TempDir("", fmt.Sprint(ts))``` is creating the temp dir. So no need to create it again. 
As part of ```os.Mkdir(context, 0750)``` is concerned, global context is misplaced somehow here i guess.

Reply:
@kadel https://github.com/openshift/odo/pull/2185#discussion_r328469771